App/Sagas/NotificationsSagas.ts

```diff
@@ -81,8 +81,6 @@ export function * notificationView (action: ActionType<typeof NotificationsActio
         if (threadData) {
           yield put(PhotoViewingAction.viewThread(threadData.thread.id))
           yield put(PhotoViewingAction.viewPhoto(notification.data_id as PhotoId))
-          yield call(
-            NavigationService.navigate, 'ViewThread', { id: threadData.thread.id, name: threadData.thread.name })
```

Comment:
Now we can just navigate straight to the comment, no need to push thread and then comment. I think once the single photo view is in, it might make more sense to navigate to it instead of comments.